libraries/abstractions/pkcs11/test/iot_test_pkcs11.c

```diff
@@ -317,13 +317,14 @@ TEST_GROUP_RUNNER( Full_PKCS11_EC )
             RUN_TEST_CASE( Full_PKCS11_EC, AFQP_Verify );
         #endif
 
-        RUN_TEST_CASE( Full_PKCS11_EC, AFQP_CreateObjectDestroyObjectCertificates );
-        RUN_TEST_CASE( Full_PKCS11_EC, AFQP_GenerateKeyPair );
-        RUN_TEST_CASE( Full_PKCS11_EC, AFQP_GetAttributeValueMultiThread );
+        #if ( pkcs11testIMPORT_PRIVATE_KEY_SUPPORT == 1 )
```

Comment:
Note to reviewers: This #if has been removed (so the 3 tests, AFQP_CreateObjectDestroyObjectCertificates, AFQP_GenerateKeyPair, AFQP_GetAttributeValueMultiThread) are run regardless of IMPORT_PRIVATE_KEY_SUPPORT) in c505df44a344e9ae8bdbf4990df9edabf47832c8